oauth/index.js

```diff
@@ -280,23 +285,6 @@ module.exports.init = function(config, logger, stats) {
         }
     };
 
-
-    return {
-
-        onrequest: function(req, res, next) {
-            if (process.env.EDGEMICRO_LOCAL == "1") {
-                debug ("MG running in local mode. Skipping OAuth");
-                next();
-            } else {
-                middleware(req, res, next);
-            }
-        },
-
-        testing: {
```

Comment:
I just added this along with testing. Why is it removed now?

Reply:
So, I guess my merge did not go quite right.

I did not realize that this was there for tests that would be left in the release code. It seemed to be a temporary sort of thing. Sorry if it made more work for you to put it back. 

A comment might have helped. But, then perhaps the member should not be added within the plugin code unless there are resources that cannot be accessed outside the module. 

I put in 'shutdown', because I could not really expose resources out of quota.  Quota has an execution path that hangs on shutdown because they have hidden timers - seems to occur frequently.  There was as yet no reason for a 'shutdown' in oauth.

So, I was considering putting the field into the object returned by the plugin, but within the test code.  So, you create the plugin, it returns the object, OBJ - say. And, then you just say OBJ.testing = fn  or OBJ.shutdown = fn.  That might be better than bringing changes into the plugin code. Perhaps I could have done that for Quota as well in some way.

Reply:
Exposing it for testing this way wasn't perfect, but I didn't have time to rewrite the entire module to make it properly testable. In any case, now I don't have much confidence in this PR as it has lost code. Perhaps we can reverse your merge and then merge properly again.